internal/statemachine/helper.go

```diff
@@ -579,6 +579,55 @@ func removePreseeding(rootfs string) (seededSnaps map[string]string, err error)
 	return seededSnaps, nil
 }
 
+// generateGerminateCmd creates the appropriate germinate command for the
+// values configured in the image definition yaml file
+func generateGerminateCmd(imageDefinition ImageDefinition) *exec.Cmd {
+	// determine the value for the seed-dist in the form of <archive>.<series>
+	seedDist := imageDefinition.Rootfs.Archive
```

Comment:
@sil2100 thoughts on renaming "archive" to "flavor"?

Reply:
My idea for the `archive:` tag was a bit different. Think of it like the Archive parameter to livefs builds. Most of the time that would be the 'ubuntu archive', but someone can build from a PPA, maybe a rebuild archive. So the values accepted there, in my view, were 'ubuntu', 'ppa:<PPA>' and - later on - maybe some alternate 'archives'.
...how useful this will be, well... this depends. I'd default to 'ubuntu' and later, maybe, consider removing it altogether. The `extra-ppas:` notion should basically be sufficient if one needs to use PPAs. This field was put in place to a more forward-facing concept, like when one has partial archives (see `ubuntu-rtm`), rebuild archives or anything that would potentially be non-ubuntu. But for now it wouldn't be relevant, really.

It's not about flavors, if anything. Flavors are a bit of an abstract idea.

That being said, we do seem to be missing an argument to the `seed:` method - seed-dist. Maybe that should be called `flavor` indeed?

Reply:
I see, I had understood it to be the `seed-dist` which is why I implemented it this way. I'll leave it now with a default of `ubuntu` and add a `flavor` key/value pair that also defaults to `ubuntu`.